project-base/app/config/graphql/types/ModelType/Slider/SliderItem.types.yaml

```
@@ -23,6 +23,15 @@ SliderItem:
             gtmCreative:
                 type: "String"
                 description: "GTM creative"
+            description:
+                type: "String!"
```

```suggestion
                type: "String"
```
the description is nullable on the entity